test/types/void/void-variable-use-3.good

```diff
@@ -1 +1 @@
-void-variable-use-3.chpl:5: error: cannot use the value of a void variable
+
```

Comment:
I was wondering what happened if a void is written out.  Looks like nothing?  Does this seem more correct than writing 'void' or something like that?  (I'm honestly not sure... can see downsides to both approaches, though potential power in what you've done here...).